local-volume/README.md

```diff
@@ -224,13 +224,27 @@ go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:LocalPersi
 [GCE Alpha](https://k8s-testgrid.appspot.com/sig-storage#gce-alpha)
 [GCE GCI Alpha](https://k8s-testgrid.appspot.com/sig-storage#gci-gce-alpha)
 
+
+## Requirements
+
+* The local-volume plugin expects paths to be stable, including across 
+  reboots and when disks are added or removed.
+
 ## Best Practices
 
 * For IO isolation, a whole disk per volume is recommended
 * For capacity isolation, separate partitions per volume is recommended
 * Avoid recreating nodes with the same node name while there are still old PVs
   with that node's affinity specified. Otherwise, the system could think that
   the new node contains the old PVs.
+* For volumes with a filesystem, its recommended to utilize their UUID (e.g.
+  the output from `ls -l /dev/disk/by-uuid`) both in fstab mounting logic and
+  in the directory name representing that volume. This practice ensures
```

Comment:
The mount directory name doesn't need to use the uuid

Reply:
The mount directory name doesn't need to use the UUID. If the directory name does contain the UUID, then it would not be confused with any other volume as the directory name is used while generating the PV name. 

This practice would help avoid the situation you've raised where the same PV name is used for two different volumes, when the directory name and node name are the same even if the volume is different. 

Would you like me to remove this recommendation?

Reply:
I agree, let's leave it.